frontend/admin/src/js/components/poolCandidate/PoolCandidateTableFilterDialog.tsx

```diff
@@ -120,6 +121,18 @@ const PoolCandidateTableFilterDialog = ({
           }}
         >
           <div data-h2-flex-grid="base(flex-start, x1, x.5)">
+            <div data-h2-flex-item="base(1of1) p-tablet(1of2) laptop(3of5)">
+              <MultiSelectFieldV2
+                id="pools"
+                label={formatMessage({
+                  defaultMessage: "Pools",
+                  id: "mjyHeP",
+                })}
+                options={optionsData.pools}
+                isLoading={rawGraphqlResults.pools.fetching}
+                isDisabled
+              />
+            </div>
```

Comment:
I'm happy this selection is added in - I'm not actually sure it should be disabled. Unless there's a technical reason, why should admins be barred from changing this?

Reply:
My initial thought was to eliminate the possibility of the admin from accidentally changing the pool and getting candidates not associated with what they are looking for. However, now I think your right it's probably better if they have that freedom to change it.

Reply:
@tristan-orourke maybe we should ask DCMO what filters they would like to see on the PoolCandidateTableFilterDialog before deploying on Thursday?

Reply:
this will definitely be something to keep an eye on for Organizations, preventing managers from viewing candidates of pools they shouldn't have access to